test/functional/autocmd/termclose_spec.lua

```diff
@@ -22,14 +20,14 @@ describe('TermClose event', function()
   end)
 
   it('triggers when long-running terminal job gets stopped', function()
-    nvim('set_option', 'shell', 'sh')
```

Comment:
After removing this line, this test is a duplicate of the `triggers when fast-exiting terminal job stops` test?

Reply:
Hmm, do the git logs reveal what happened?

Reply:
It was added in https://github.com/neovim/neovim/pull/6823.

Reply:
How about using powershell without the `-NoProfile` flag?